pixi.toml

```diff
@@ -102,11 +105,12 @@ install-polars-nightly = "pip install --pre --no-deps --upgrade --only-binary :a
 
 [environments]
 build = ["build"]
-default = ["dev", "lint", "test"]
+default = ["dev", "lint", "test", "optionals"]
 docs = ["docs"]
 lint = { features = ["lint"], no-default-feature = true }
 nightly = ["nightly", "test"]
 py310 = ["py310", "test"]
 py311 = ["py311", "test"]
 py312 = ["py312", "test"]
 py313 = ["py313", "test"]
+py313-optionals = ["py313", "test", "optionals"]
```

Comment:
I considered running also the environments twice, but that felt like overkill to me.